settings/settings.php

```diff
@@ -242,11 +242,14 @@ public function handle_actions( $action ) {
 
 				if ( $nolang = $this->model->get_objects_with_no_lang() ) {
 					$lang = $this->model->get_language( $this->options['default_lang'] );
-					if ( ! empty( $nolang['posts'] ) ) {
-						$this->model->post->set_language_in_mass( $nolang['posts'], $lang );
-					}
-					if ( ! empty( $nolang['terms'] ) ) {
-						$this->model->term->set_language_in_mass( $nolang['terms'], $lang );
+
+					if ( ! empty( $lang ) ) {
```

Comment:
I believe there is still one edge case where it could fail, but so edgy that it can be discarded imho:
- This action (`content-default-lang`) is available only when the "There are posts without language" notice is displayed.
- In this notice, [the default language option is checked](https://github.com/polylang/polylang/blob/db0de3d90ae5053e16fc151c5aa80f0d0124ac2a/settings/settings.php#L347), but not if the language itself exists.
- So, at this point in `settings.php` we could have:
    - Available languages: 'en', 'fr'.
    - Default language option: 'de'.

Like I said, too edgy to be considered, I'll do the change 👍

Reply:
Done in [6b71ef2](https://github.com/polylang/polylang/pull/1207/commits/6b71ef2a2d15d50793e8ac2ba85885a4f81d810e).